static/js/resize.js

```diff
@@ -23,7 +23,7 @@ function size_blocks(blocks, usable_height) {
     _.each(blocks, function (block) {
         var ratio = block.real_height / sum_height;
         ratio = confine_to_range(0.05, ratio, 0.85);
```

Comment:
We don't need to change ratio. We just need to change the value we pass to <code>lo</code> in the function <code>confine_to_range(lo,val,high)</code>.